lib/bridge/strain.rb

```diff
@@ -36,6 +36,21 @@ def self.minors
       [Club, Diamond]
     end
 
+    @mapping = {}
+    %w(Clubs Diamonds Hearts Spades NoTrumps
```

Comment:
@timheilman which book of Robert C. Martin's are you reading? Based on the tidbits you've brought up I suspect I'd learn a lot reading it.